Sources/Common/SwiftSupport.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  Created by David Jennes on 7/10/18.
+//  Copyright © 2018 IBAnimatable. All rights reserved.
+//
+
+import UIKit
+
+/// Swift < 4.2 support
+#if !(swift(>=4.2))
+enum CAMediaTimingFunctionName {
```

Comment:
Great idea 👍

Reply:
This could help me with another project. Thanks for the trick @djbe 💯